src/Markdig/Extensions/Emoji/EmojiMapping.cs

```diff
@@ -0,0 +1,1784 @@
+                {":norfolk_island:","🇳🇫"},
+                {":nigeria:","🇳🇬"},
+                {":nicaragua:","🇳🇮"},
+                {":netherlands:","🇳🇱"},
+                {":norway:","🇳🇴"},
+                {":nepal:","🇳🇵"},
+                {":nauru:","🇳🇷"},
+                {":niue:","🇳🇺"},
+                {":new_zealand:","🇳🇿"},
+                {":oman:","🇴🇲"},
+                {":panama:","🇵🇦"},
+                {":peru:","🇵🇪"},
+                {":french_polynesia:","🇵🇫"},
+                {":papua_new_guinea:","🇵🇬"},
+                {":philippines:","🇵🇭"},
+                {":pakistan:","🇵🇰"},
+                {":poland:","🇵🇱"},
+                {":st_pierre_miquelon:","🇵🇲"},
+                {":pitcairn_islands:","🇵🇳"},
+                {":puerto_rico:","🇵🇷"},
+                {":palestinian_territories:","🇵🇸"},
+                {":portugal:","🇵🇹"},
+                {":palau:","🇵🇼"},
+                {":paraguay:","🇵🇾"},
+                {":qatar:","🇶🇦"},
+                {":reunion:","🇷🇪"},
+                {":romania:","🇷🇴"},
+                {":serbia:","🇷🇸"},
+                {":rwanda:","🇷🇼"},
+                {":saudi_arabia:","🇸🇦"},
+                {":solomon_islands:","🇸🇧"},
+                {":seychelles:","🇸🇨"},
+                {":sudan:","🇸🇩"},
+                {":sweden:","🇸🇪"},
+                {":singapore:","🇸🇬"},
+                {":st_helena:","🇸🇭"},
+                {":slovenia:","🇸🇮"},
+                {":svalbard_jan_mayen:","🇸🇯"},
+                {":slovakia:","🇸🇰"},
+                {":sierra_leone:","🇸🇱"},
+                {":san_marino:","🇸🇲"},
+                {":senegal:","🇸🇳"},
+                {":somalia:","🇸🇴"},
+                {":suriname:","🇸🇷"},
+                {":south_sudan:","🇸🇸"},
+                {":sao_tome_principe:","🇸🇹"},
+                {":el_salvador:","🇸🇻"},
+                {":sint_maarten:","🇸🇽"},
+                {":syria:","🇸🇾"},
+                {":swaziland:","🇸🇿"},
+                {":tristan_da_cunha:","🇹🇦"},
+                {":turks_caicos_islands:","🇹🇨"},
+                {":chad:","🇹🇩"},
+                {":french_southern_territories:","🇹🇫"},
+                {":togo:","🇹🇬"},
+                {":thailand:","🇹🇭"},
+                {":tajikistan:","🇹🇯"},
+                {":tokelau:","🇹🇰"},
+                {":timor_leste:","🇹🇱"},
+                {":turkmenistan:","🇹🇲"},
+                {":tunisia:","🇹🇳"},
+                {":tonga:","🇹🇴"},
+                {":tr:","🇹🇷"},
+                {":trinidad_tobago:","🇹🇹"},
+                {":tuvalu:","🇹🇻"},
+                {":taiwan:","🇹🇼"},
+                {":tanzania:","🇹🇿"},
+                {":ukraine:","🇺🇦"},
+                {":uganda:","🇺🇬"},
+                {":us_outlying_islands:","🇺🇲"},
+                {":united_nations:","🇺🇳"},
+                {":uruguay:","🇺🇾"},
+                {":uzbekistan:","🇺🇿"},
+                {":vatican_city:","🇻🇦"},
+                {":st_vincent_grenadines:","🇻🇨"},
+         
```

Comment:
It's not obvious, but the `PrefixTree.Add()` will throw for null or empty keys already

Reply:
Oh ok. Do you prefer I remove this line then? It's not totally useless as it outputs a nicer message, with the name of the parameter in question.

Reply:
We can keep it, just move it up so that it will be hit first